agent/otel/otlpmqttexporter/otlp.go

```diff
@@ -99,6 +101,12 @@ func (e *exporter) pushTraces(_ context.Context, _ ptrace.Traces) error {
 	return fmt.Errorf("not implemented")
 }
 
+// extract policy from metrics Request
+func extractPolicy(metricsRequest pmetricotlp.Request) string {
+	p, _ := metricsRequest.Metrics().ResourceMetrics().At(0).ScopeMetrics().At(0).Metrics().At(1).Gauge().DataPoints().At(0).Attributes().Get("policy")
```

Comment:
It seems this assumes the first metric is a Gauge, I don't think this will work.

Reply:
I fixed looking for gauge metrics for each handler we have